### ISSUE TITLE
display-popup -h/-w with no value results in "[server exited unexpectedly]"

### ISSUE THREAD

**Issue by sltrash** [tmux/tmux#3664]
### Issue description

Hello:-)

While reading the changelog and trying the -B option to the display-popup command I mistakenly typed -h instead of -B which crashed the tmux server:

```
$ tmux display-popup -h
[server exited unexpectedly]
```

Obviously this is not intented! :-O

- The same happens with the '-w' option
-  -h and -w expect an additionnal parameter (height & width respectively)
- no other display-popup option seem to suffer the same problem

Some additionnal tests
```
$ tmux display-popup -h r
height invalid
$ tmux display-popup -h ''
height invalid
$ tmux display-popup -h 
[server exited unexpectedly]
```

Many thanks for everthing about tmux!

slt

### Required information

Please provide the following information:

* tmux version {3.3a}
* Platform (Linux unknown)
* $TERM inside and outside of tmux (tmux-256color / xterm-256color).
* Logs from tmux (server exited unexpectedly).

**Comment by gierens** [comment]
I can reproduce this with my installed tmux 3.3a ... but when running a new session with a fresh build from `master` the `display-popup -h` responds with `-h expects an argument` as expected, so this seems to have been fixed since 3.3a.

**Comment by nicm** [comment]
This has already been fixed.
